model/accounts_mgmt/v1/notify_details_resource.model

```diff
@@ -0,0 +1,24 @@
+/*
+Copyright (c) 2024 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Manages the notify endpoint.
+resource NotifyDetails {
+	// Post Notification details about user related to subscription/cluster via email and get the data associated with it.
+	method Post {
```

Comment:
method should be `Add`

Reply:
can't use `Add` as we need to use two separate `in` and `out` structs

Reply:
Ack